README.md

```diff
@@ -201,6 +212,14 @@ The following documentation is available:
 <a name="known-issues"></a>
 ## Known Issues
 
+<a name="bugs"></a>
+### Bugs
+
+For a list of current bugs, see the [release notes](https://github.com/minishift/minishift/releases).
```

Comment:
-1, the release notes don't list the current bugs. It is the issue tracker which has the current bugs. The release notes, refer to bugs/issues fixed.